import_logs.py

```diff
@@ -1655,6 +1671,30 @@ def _record_hits(self, hits):
 
                 if config.options.debug_tracker:
                     logging.debug('tracker response:\n%s' % response)
+
+                # check for invalid requests
+                try:
+                    response = json.loads(response)
+                except:
+                    logging.info("bulk tracking returned invalid JSON")
+
+                    # don't display the tracker response if we're debugging the tracker.
+                    # debug tracker output will always break the normal JSON output.
+                    if not config.options.debug_tracker:
+                        logging.info("tracker response:\n%s" % response)
+                
+                if ('invalid_indices' in response and isinstance(response['invalid_indices'], list) and
```

Comment:
Would it trigger an exception if response is eg `null` because no json is returned? Not so familiar with Python anymore. Been a few years since I last used it.